### DISCUSSION TITLE
[API Request] getRated (equivalent to getStarred but for ratings)

### DISCUSSION THREAD

**Discussion by jmbannon** [opensubsonic/open-subsonic-api#112] [API requests]
### Type of change

API Clarification

### Proposal description

Would implement the getter for `/setRating` (https://opensubsonic.netlify.app/docs/endpoints/setrating/), to get all tracks that have any kind of rating.

This would benefit users who prefer the granularity of ratings versus starred

### Backward compatibility impact

n/a

### Backward compatibility

- [X] No backward compatibility impact.

### API details

Equivalent to https://opensubsonic.netlify.app/docs/endpoints/getstarred2/

### Security impacts

None

### Potential issues

The order in which they are returned could be debated

### Alternative solutions

Using stars, but again, less granularity.

**Comment by Tolriq** [comment]
This is planned in the new endpoints that will allow filtering.
